airbyte-integrations/connectors/source-datadog/source_datadog/spec.yaml

```diff
@@ -4,35 +4,43 @@ connectionSpecification:
   title: Datadog Source Spec
   type: object
   required:
+    - site
```

Comment:
@bartveenstra we can't add a required param after the fact since it will break existing connections

Reply:
I'll remove it from required, but I wonder how it would work when you do need to introduce required parameters.